packages/postcss-svgo/package.json

```
@@ -28,16 +28,16 @@
   "repository": "cssnano/cssnano",
   "dependencies": {
     "postcss-value-parser": "^4.2.0",
-    "svgo": "^2.7.0"
+    "svgo": "^3.0.1"
```

If we bump this to 3.0.2 we can get rid of the `@types/csso` package as well.
```suggestion
    "svgo": "^3.0.2"
```

---

Updated! :rocket: